src/stack.py

```diff
@@ -0,0 +1,26 @@
+"""implments the stack"""
+from linked_list import LinkedList
+
+
+class Stack(object):
+    """Is the start of the stack object class."""
+
+    def __init__(self, itter=None):
+        """Creates the new stack using the linked list."""
+        self._linked_list = LinkedList(itter)
+        self.head = self._linked_list.head
```

Comment:
Not a ding on points, but why do you even need a head here? All you're doing is composing a data structure from a linked list, so why not just keep track of what's on the linked list?